presto-main/src/test/java/com/facebook/presto/execution/TestPrepareTask.java

```diff
@@ -72,11 +77,25 @@ public void testPrepareNameExists()
         assertEquals(statements, ImmutableMap.of("my_query", "SELECT *\nFROM\n  foo\n"));
     }
 
-    private Map<String, String> executePrepare(String statementName, Query query, String sqlString, Session session)
+    @Test
+    public void testPrepareInvalidStatement()
+    {
+        Statement statement = new Execute("foo");
+        String sqlString = "PREPARE my_query FROM EXECUTE foo";
+        try {
+            executePrepare("my_query", statement, sqlString, TEST_SESSION);
```

Comment:
Deleting comment since I see this is consistent with other places in this class, and consistency is better